pombola/writeinpublic/tests.py

```diff
@@ -126,6 +180,43 @@ def test_get_messages(self, m):
         self.assertEqual(messages[0].subject, 'Test message')
         self.assertEqual(messages[1].subject, 'Another test message')
 
+    def test_get_person(self, m):
+        m.get('/api/v1/person/', json=person_json)
+        person = Person.objects.create(
+            name="Jimmy Stewart",
+            slug="jimmy-stewart")
+        person_result = self.writeinpublic.get_person(person)
+        last_request = m._adapter.last_request
+        expected_qs = {
+            'username': ['test'],
+            'api_key': ['123'],
+            'format': ['json'],
+            'identifiers__scheme': ['popolo:person'],
+            'identifiers__identifier': [str(person.id)],
+        }
+        self.assertEqual(last_request.qs, expected_qs)
+        self.assertEqual(len(person_result), 1)
+        self.assertEqual(person_result[0].email, person_json['objects'][0]['email'])
+
+    def test_get_person_is_contactable(self, m):
+        m.get('/api/v1/person/', json=person_json)
+        person = Person.objects.create(
+            name="Jimmy Stewart",
+            slug="jimmy-stewart")
+        result = self.writeinpublic.get_person_is_contactable(person)
+        last_request = m._adapter.last_request
+        expected_qs = {
+            'username': ['test'],
+            'api_key': ['123'],
+            'format': ['json'],
+            'identifiers__scheme': ['popolo:person'],
+            'identifiers__identifier': [str(person.id)],
+            'has_contacts': ['true'],
+            'instance_id': ['42']
```

Comment:
in theory you can check multiple contacts at once to reduce the number of queries because you can pass a param multiple times, but that's a premature optimisation